src/app/components/LiveLabel/index.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  liveLabel: ({ palette, fontVariants }: Theme) =>
+    css({
+      color: palette.POSTBOX,
+      display: 'inline-block',
+      ...fontVariants.sansBold,
+    }),
+  textRtl: ({ spacings }: Theme) =>
+    css({
+      marginLeft: `${spacings.FULL}rem`,
+    }),
+  textLtr: ({ spacings }: Theme) =>
+    css({
+      marginRight: `${spacings.FULL}rem`,
+    }),
```

Comment:
We could use the [margin-inline](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline) property here to let the browser work out whether to apply left/right margins for the given direction